INTITA.xcodeproj/xcshareddata/xcschemes/INTITA.xcscheme

```diff
@@ -45,7 +45,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
-      language = "en"
+      language = "uk"
```

Comment:
This line will change the default language of the application.
Keep in mind - you don't have to push such changes if it is not a part of the task. It might be discard during the checking the changes.  If you want to left it, just notify the team to avoid any conflict or misunderstanding.

Reply:
It had changed to "en".